CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -113,12 +113,16 @@ local function UpdateLayout()
 		end
 	end
 	
-	width = #ToolsList * ICON_SPACING
-	height = ICON_SIZE + HEALTHBAR_SPACE
+	if #ToolsList == 0 then
```

Comment:
Yeah, we're sharing the panel. If there are no tools, it just shrinks to the healthbar. We don't want it oversized since it can block other raycasts (for now.)